go/vt/sqlparser/keywords.go

```diff
@@ -318,6 +318,13 @@ var keywords = []keyword{
 	{"join", JOIN},
 	{"json", JSON},
 	{"json_table", JSON_TABLE},
+	{"json_contains", JSON_CONTAINS},
+	{"json_contains_path", JSON_CONTAINS_PATH},
+	{"json_extract", JSON_EXTRACT},
+	{"json_keys", JSON_KEYS},
+	{"json_overlaps", JSON_OVERLAPS},
+	{"json_search", JSON_SEARCH},
+	{"json_value", JSON_VALUE},
 	{"json_pretty", JSON_PRETTY},
 	{"json_storage_size", JSON_STORAGE_SIZE},
 	{"json_storage_free", JSON_STORAGE_FREE},
```

Comment:
This should also be alphabetically sorted

Reply:
> We should also add tests for the shorthands `->` and `->>`

There were some tests already present so I didn't add them.

Reply:
This might not be the place for the tests, but there was a regression in v13 #9951

Reply:
@derekperkins I plan to look at the issue before merging this PR today. I already tried running the query through the parser mentioned in #9951 and it passed. I think the issue might be in the planner, but I'll know more today and update on the issue